src/ui/public/index_patterns/index_patterns.js

```diff
@@ -41,6 +41,7 @@ export function IndexPatternsProvider(Notifier, Private) {
   self.cache = patternCache;
   self.getIds = getProvider('id');
   self.getTitles = getProvider('attributes.title');
+  self.getIndexPatterns = getProvider();
```

Comment:
Great suggestion @spalger. Like @Stacey-Gammon, I'm not a fan of `getSavedObjects`. Based on the other calls, I'd honestly expect `self.get` to return the entire object.